app/src/main/java/org/mozilla/fenix/collections/CollectionCreationView.kt

```diff
@@ -73,10 +73,6 @@ class CollectionCreationView(
             view.context,
             R.layout.component_collection_creation_select_collection
         )
-        nameCollectionConstraints.clone(
-            view.context,
-            R.layout.component_collection_creation_name_collection
```

Comment:
I notice this file contains two ConstraintLayouts and is being added to a ConstraintLayout: I wonder if the performance problems would go away with less complexity if we simplified this layout and the other nested layouts added rather than deferring initialization. That being said, to do that, we'd have to change the top-level layout to use `<include>` and each included layout to have a top-level `<merge>`.